### DISCUSSION TITLE
Getting better at catalogues of stuff using markdown

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#923] [General]
I found this tutorial https://portaljs.org/blog/create-a-simple-catalog-of-anything-using-markdown very helpful to create a simple table view of some ecosystem mapping research profiles created and edited in Obsidian.

![Screenshot 2023-11-27 at 17 11 12](https://github.com/life-itself/community/assets/97066487/236d70b7-6bda-4e9a-9404-fa98910dbaab)

Now I have some questions. Collecting them here as these might be topics for future tutorials/how-to guides to address. 
- How do I create a dataview table with a subset of files from a folder. e.g. with the above tutorial I know how to create a table that displays all of the files in a folder. How do I create a table that shows only all of the files in a folder that meet a certain condition? (e.g. if I want to create a table that shows all of the files in a folder where the cohere property is true)

**Comment by martinprogrammer** [comment]
@catherinet1  I used this:  
<img width="659" alt="image" src="https://github.com/life-itself/community/assets/6898082/c438cc82-af80-4cb2-95ff-c4f54f772d67">

and in the file you'll have something like:
`---`
`name: London`
`---`

**Reply by catherinet1** [reply]
ah, thanks @martinprogrammer ! 🙏

**Comment by martinprogrammer** [comment]
@catherinet1 
reference to querying language for the dataview
https://blacksmithgu.github.io/obsidian-dataview/queries/dql-js-inline/
